docs/source/app-dev/app-arch.rst

```diff
@@ -1,295 +1,121 @@
 .. Copyright (c) 2020 The DAML Authors. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-Application architecture guide
-#########################################
+Application architecture
+########################
 
-This document is a guide to building applications that interact with a DA ledger deployment (the 'ledger'). It:
+This section describes our recommended design of a full-stack DAML application.
 
-- describes the characteristics of the ledger API, how this affects the way an application is built (the 'application architecture'), and why it is important to understand this when building applications
-- describes the resources in the SDK to help with this task
-- gives some guidelines to help you build correct, performant, and maintainable applications using all of the supported languages
+.. image:: ./recommended_architecture.svg
 
-Categories of application
-*************************
+The above image shows the recommended architecture. If you are working with a different tech stack DAML SDK also supports :ref:`Java <java-bindings>` and :ref:`Scala
+<scala-bindings>` bindings as well as a direct :ref:`gRPC <grpc>` endpoint (instead of JSON API)
 
-Applications that interact with the ledger normally fall into four categories:
+To get started quickly with the recommended application architecture clone the```
+``create-daml-app`` application template:
 
-.. list-table:: Categories of application
-   :header-rows: 1
+.. code-block:: bash
 
-   * - Category
-     - Receives transactions?
-     - Sends commands?
-     - Example
-   * - Source
-     - No
-     - Yes
-     - An injector that reads new contracts from a file and injects them into the system.
-   * - Sink
-     - Yes
-     - No
-     - A reader that pipes data from the ledger into an SQL database.
-   * - Automation
-     - Yes
-     - Yes, responding to transactions
-     - Automatic trade registration.
-   * - Interactive
-     - Yes (and displays to user)
-     - Yes, based on user input
-     - DA’s :doc:`Navigator </tools/navigator/index>`, which lets you see and interact with the ledger
+  git clone https://github.com/digital-asset/create-daml-app
 
-Additionally, applications can be written in two different styles:
+``create-daml-app`` is a small, but fully functional demo application implementing the recommended
+architecture, including
 
--  Event-driven - applications base their actions on individual ledger events only.
--  State-driven - applications base their actions on some model of all contracts active on the ledger.
+- DAML React libraries
+- quick iteration against the :ref:`DAML Ledger Sandbox <sandbox-manual>`.
+- authentication
```

Comment:
Maybe point to the "Authentication" section on this page? I think it would work better than pointing directly to the authentication page itself

Reply:
not sure if I find a link just pointing to a paragraph within the same page to useful. I find it almost more annoying to read because I have to check where the link is going.